.pre-commit-config.yaml

```
@@ -2,4 +2,27 @@ repos:
 -   repo: https://github.com/psf/black
     rev: 19.10b0
     hooks:
-    - id: black
\ No newline at end of file
+    - id: black
+-   repo: https://github.com/asottile/seed-isort-config
+    rev: v2.1.1
+    hooks:
+    -   id: seed-isort-config
+-   repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21
+    hooks:
+    -   id: isort
+-   repo: https://github.com/Yelp/detect-secrets
+    rev: v0.14.2
```

`detect-secrets` has a v0.14.3 available
```suggestion
    rev: v0.14.3
```